### ISSUE TITLE
Looking for a C++ alternative for symbol.get_internals() (in Python API)

### ISSUE THREAD

**Issue by Bumblebee1964** [apache/incubator-mxnet#7267]
For bugs or installation issues, please provide the following information.
The more information you provide, the more likely people will be able to help you.

## Environment info
Operating System: win10

Compiler: VS2015

Package used (Python/R/Scala/Julia): c-api

MXNet version:

Or if installed from source:

MXNet commit hash (`git rev-parse HEAD`):

If you are using python package, please provide

Python version and distribution:

If you are using R package, please provide

R `sessionInfo()`:

## Error Message:
I would like to change a loaded existing network. In http://mxnet.io/how_to/finetune.html is explained how to do this using the Python API. I work with the C++ API and look for a replacement of the code line "all_layers = symbol.get_internals()" 
In the C++ API I can't find a similar function.

A code snippet reflecting the Python functionality in C++ would be very much appreciated.
## Steps to reproduce

1.
2.
3.

## What have you tried to solve it?

1.
2.
3.

**Comment by Bumblebee1964** [comment]
Used following solution (for lenet example in c++ to have 2 instead of 10 classes)
     // replace last fc layer with new fc layer with 2 outputs
     Symbol all_layers = lenet.GetInternals();
     Symbol out = all_layers["fc2_output"];
     Symbol fc3 = FullyConnected("fc3", out, fc3_w, fc3_b, 2);

     Symbol lenetNew = SoftmaxOutput("softmax", fc3, data_label);
